R/rNET-env.R

```diff
@@ -0,0 +1,36 @@
+# Environment that holds various global variables and settings for the package.
+# It is not exported and should not be directly manipulated by other packages.
+rNETenv <- new.env(parent = emptyenv())
```

Comment:
Created an environment following the same concept we have in other osp packages.